### PR TITLE
Fix codespell issues

### PR DESCRIPTION
**Pull request by mjtrangoni**
### What does this PR do?
This fixed some trivial spelling issues found by [codespell](https://github.com/codespell-project/codespell).
I think this could be added to the CI, and if you agree, I can fix the remaining spelling issues in one or more separate PR.

### What issues does this PR fix or reference?
none

### Tests written?
No, no needed

### Commits signed with GPG?
No

### CODE REVIEW

**Review by rares-pop — APPROVED** [review]

**Comment by Ch3LL** [comment]
@mjtrangoni looks like there is a merge conflict here, if you could take care of that.

Also @saltstack/team-core any thoughts on adding codespell to our test infrastructure?

**Comment by mjtrangoni** [comment]
@Ch3LL done

File: salt/modules/glance.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def get(key, default=None):
     region = get('region')
     ks_endpoint = get('endpoint', 'http://127.0.0.1:9292/')
     g_endpoint_url = __salt__['keystone.endpoint_get']('glance', profile)
-    # The trailing 'v2' causes URLs like thise one:
+    # The trailing 'v2' causes URLs like these one:
```
Comment (twangboy) [inline salt/modules/glance.py]: this

File: salt/modules/highstate_doc.py [hunk 2]
```diff
@@ -487,16 +486,16 @@ def _blacklist_filter(s, config):
...
-    return proccessed lowstate data that was not blacklisted
+    return processed lowstate data that was not blacklisted
 
     render_module_function is used to provide your own.
     defaults to from_lowstate
     '''
     states = []
     config = _get_config(**kwargs)
-    proccesser = config.get('proccesser')
+    processor = config.get('processor')
```
Comment (twangboy) [inline salt/modules/highstate_doc.py]: This whole section makes me a little nervous. Have all references to this function also been renamed?

File: salt/modules/highstate_doc.py [hunk 3]
```diff
@@ -590,16 +589,16 @@ def _format_markdown_requisite(state, stateid, makelink=True):
         return ' * `{0}`\n'.format(fmt_id)
 
 
-def proccesser_markdown(lowstate_item, config, **kwargs):
+def processor_markdown(lowstate_item, config, **kwargs):
```
Comment (twangboy) [inline salt/modules/highstate_doc.py]: Same thing here...

File: salt/modules/localemod.py [hunk 4]
```diff
@@ -43,9 +43,9 @@ def _parse_dbus_locale():
     Get the 'System Locale' parameters from dbus
     '''
     bus = dbus.SystemBus()
-    localed = bus.get_object('org.freedesktop.locale1',
+    localedbus = bus.get_object('org.freedesktop.locale1',
                              '/org/freedesktop/locale1')
-    properties = dbus.Interface(localed, 'org.freedesktop.DBus.Properties')
+    properties = dbus.Interface(localedbus, 'org.freedesktop.DBus.Properties')
```
Comment (twangboy) [inline salt/modules/localemod.py]: `locale_dbus` or `locale_bus`... or just `locale` maybe

File: salt/modules/victorops.py [hunk 5]
```diff
@@ -200,7 +200,7 @@ def create_event(message_type=None, routing_key='everybody', **kwargs):
             if isinstance(kwargs[kwarg], keyword_args[kwarg]):
                 data[kwarg] = kwargs[kwarg]
             else:
-                # Should this faile on the wrong type.
+                # Should this failed on the wrong type.
```
Comment (twangboy) [inline salt/modules/victorops.py]: fail

File: salt/modules/win_status.py [hunk 6]
```diff
@@ -379,7 +379,7 @@ def procs(count=False):
     if count:
         return len(processes)
 
-    #a propper run of the function, creating a nonsensically long out put.
+    #a proper run of the function, creating a nonsensically long out put.
```
Comment (twangboy) [inline salt/modules/win_status.py]: output

File: salt/modules/zpool.py [hunk 7]
```diff
@@ -1557,7 +1557,7 @@ def reopen(zpool):
         salt '*' zpool.reopen myzpool
 
     '''
-    ## reopen all devices fro pool
+    ## reopen all devices for a pool
```
Comment (twangboy) [inline salt/modules/zpool.py]: from a

**Review by twangboy — CHANGES_REQUESTED** [review]

**Comment by waynew** [comment]
I'm 👍 on adding codespell to testing

**Comment by mjtrangoni** [comment]
@twangboy PTAL

**Comment by sagetherage** [comment]
@mjtrangoni can you please rebase this against the master branch?

**Comment by sagetherage** [comment]
@mjtrangoni can you please rebase this against the master branch?

**Comment by sagetherage** [comment]
Reviewed in Grooming 2021-MAY-03 closing

**Pull request closed without merging**